examples/monaco-editor-webpack-plugin/webpack.config.js

```diff
@@ -38,13 +32,14 @@ export default {
       customLanguages: [
         {
           label: 'yaml',
+          // eslint-disable-next-line prettier/prettier
```

Comment:
Can you remove this line and format the code?

Reply:
Done